IntroToCSharp/Pages/Activities/Math.razor

```diff
@@ -0,0 +1,143 @@
+@inherits CaptainCoder.Page
+@page "/activities/math"
+
+<Section Title="Introduction">
+    <MudText>At the heart of all computers and computer programs is math. Most modern-day computers are cable of doing
+        math on an extremely large-scale,
+        but today we are going to learn about how to interpret and implement basic arithmetic within our own programs. A
```

Comment:
`Today, modern computers are capable of doing thousands of mathematical computations per minute. In this lesson we ...`